packages/tools/pie-css/README.md

```diff
@@ -20,12 +20,16 @@
     3. Sass /SCSS
     4. Native HTML
 4. [What's included in the `pie-css` base stylesheet](#whats-included-in-the-pie-css-base-stylesheet)
```

Comment:
I don't think this is accurate anymore? The animations are not in the base stylesheet from what I can see

Reply:
They are imported [here](https://github.com/justeattakeaway/pie/pull/2435/files#diff-3dc2c07456fce01daab73f3d0b09d023bb88ccb1da6b1d076142d4dfad203394L3) as part of the main entry 👀